fetch.bs

```diff
@@ -6841,14 +6799,14 @@ partial interface mixin WindowOrWorkerGlobalScope {
 
 <p>The
 <dfn id=dom-global-fetch method for=WindowOrWorkerGlobalScope><code>fetch(<var>input</var>, <var>init</var>)</code></dfn>
-method, must run these steps:
+method steps are:
 
 <ol>
  <li><p>Let <var>p</var> be a new promise.
```

Comment:
This could be `[=a new promise=]`, although then you should specify the realm; I'd be curious whether it's relevant or current in implementations...